### ISSUE TITLE
Compilation console color all red is "rude"

### ISSUE THREAD

**Issue by Geobert** [intellij-rust/intellij-rust#1261]
Minor issue here, but I found having all the compiler output red feels "rude". 
I sometime believe that I have errors but it's only warnings (mainly because my function has just been written and not used yet).

Can we have the default colours from the compiler?

**Comment by matklad** [comment]
>Can we have the default colours from the compiler?

This is exactly how it is supposed to work :) Could you give us a screenshot of how compiler output looks like now? What operation system are you using? 

cc @alygin

**Comment by Geobert** [comment]
Win 10, IntelliJ 2017.1.3, Rust 0.1.0.1991
![image](https://cloud.githubusercontent.com/assets/72570/26325318/5765cb76-3f2e-11e7-8140-9cb15475f582.png)

**Comment by alygin** [comment]
The problem here is that Rust only has two options for colored output on Windows: it uses Win API calls by default (not supported by IntelliJ platform, as far as I know) and the `msys` mode that produced broken output [last time I checked](https://github.com/AtomBuild/atom-build-cargo/issues/56#issuecomment-239621138). That's why [we decided](https://github.com/intellij-rust/intellij-rust/issues/646#issuecomment-263497400) to only use colors on Linux and Mac.

I'll have a look at how it works now. There were changes in the `term` crate since then, so maybe it's already fixed and we can have colored output on Windows (though [the issue](https://github.com/Stebalien/term/issues/58) is still open).
